packages/rpclib/rpclib.5.9.0/opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "alcotest" {with-test & < "1.0.0"}
   "jbuilder"
   "cmdliner"
```

Comment:
```suggestion
  "cmdliner" {>= "1.0.0"}
```
To solve the unbound value `const`. I don’t know if that is the bet possible but should work

Reply:
`const` was added in `0.9.8`